app/javascript/src/components/PlayerInvalid.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+
+export const PlayerInvalid: React.SFC<{}> = () => {
+  return (
+    <main>
+      Your story's all messed up. You probably deleted the start scene.
+      <br />
+      <br />
+      <b>Shouldn't have done that.</b>
```

Comment:
`<p>` and `<strong>` here maybe?

Reply:
This markup is rough... Yes.

Reply:
you talking trash about my markup???

Reply:
@efatsi it was a group effort. Let's be trash friends.

Reply:
Ack. I didn't do this... Sorry @leobauza.

PR Shortly.